pdchaoskit/provider.py

```diff
@@ -0,0 +1,17 @@
+from chaoslib.settings import get_loaded_settings
+from logzero import logger
+
+from pdchaoskit.api.scripts import get_script
+from pdchaoskit.api.session import client_session
+
+
+def script_content(id: str) -> str:
```

Comment:
What if we call this module "scripts" and and this method get_content() or get_script_content().

So very similar what you did in api module but this one is already handling you the api call

Provider may be a bit too generic. Checkout how other modules are called in chaostoolkit.

Reply:
Good idea. Done